server/index.js

```diff
@@ -66,8 +64,8 @@ app.get('/api/greeting', (req, res) => {
   res.send(JSON.stringify({ greeting: `Hello ${name}!` }));
 });
 
-// proxy to the webapp
-app.get("*", proxy(webappUrl));
+// serve web application
+app.use(express.static('build'));
```

Comment:
I think we need both no? one for static files like images, and one for react-router with `*` (if we will have in the future)